### PR TITLE
Add Matrix to dashboard

### PR DESCRIPTION
**Pull request by irgendwr**
Adds Matrix to the dashboard.

French translation by @HansiMcKlaus (8simke)

### CODE REVIEW

**Comment by irgendwr** [comment]
unfortunately I can't request reviewers so I'll tag @timonegk instead :p

File: locale/de_DE/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1666,6 +1666,14 @@ msgstr "E-Mail wurde verworfen."
 msgid "Mailinglist settings saved."
 msgstr "Mailinglisteneinstellungen gespeichert."
 
+#: mafiasi/matrix/apps.py:10
+msgid "Matrix"
+msgstr "Matrix"
+
+#: mafiasi/matrix/apps.py:11
+msgid "Matrix provides decentralized messaging and VoIP with E2E encryption."
+msgstr "Matrix ermöglicht dezentrales chatten und VoIP mit E2E Verschlüsselung."
```
Comment (timonegk) [inline locale/de_DE/LC_MESSAGES/django.po:1675]:
```suggestion
msgstr "Matrix ermöglicht dezentrales Chatten und VoIP mit Ende-zu-Ende-Verschlüsselung."
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1692,6 +1692,14 @@ msgstr "Courrier a été écartée."
 msgid "Mailinglist settings saved."
 msgstr "Mailing paramètres sauvegardés."
 
+#: mafiasi/matrix/apps.py:10
+msgid "Matrix"
+msgstr "Matrix"
+
+#: mafiasi/matrix/apps.py:11
+msgid "Matrix provides decentralized messaging and VoIP with E2E encryption."
+msgstr "Matrix provide messagerie décentralisé et VoIP avec E2E cryptage"
```
Comment (timonegk) [inline locale/fr_FR/LC_MESSAGES/django.po:1701]:
```suggestion
msgstr "Matrix fournit messagerie décentralisée et VoIP avec chiffrement de bout en bout."
```

**Review by timonegk — CHANGES_REQUESTED** [review]
Looks good apart from minor translation issues.

**Comment by irgendwr** [comment]
Thanks Timon, I added your suggestions

**Review by timonegk — APPROVED** [review]

**Review by ftsell — APPROVED** [review]

**Pull request merged**